san/available_metrics_for_slug.py

```diff
@@ -0,0 +1,13 @@
+from .graphql import execute_gql
```

Comment:
I think `available_metrics_for_slug ` that belongs to `available_metrics.py`.
Also you should add it into the README below `Available Metrics` so users know about it.